app/process/watcher/consensus-message/consensus-message.go

```diff
@@ -88,15 +90,29 @@ func (ctw ConsensusTopicWatcher) subscribeToTopic(q *queue.Queue) {
 	log.Infof("Started Consensus Message Watcher for topic [%s]\n", ctw.topicID)
 	unprocessedMessages, err := ctw.client.GetUnprocessedMessagesAfterTimestamp(ctw.topicID, milestoneTimestamp)
 	if err != nil {
+		log.Errorf("Could not get unprocessed messages after timestamp [%s]", milestoneTimestamp)
 		log.Fatal(err)
 	}
 	ctw.started = true
+	log.Println(unprocessedMessages)
```

Comment:
You should remove this log or add more concrete info. F.e `Found N unprocessed messages. Processing now` or something like that